components/host-dashboard/AddFundsModal.js

```diff
@@ -352,6 +352,17 @@ const AddFundsModal = ({ host, collective, ...props }) => {
               source: resultOrder.fromAccount,
               tier: resultOrder.tier,
             });
+            /*
+             * Since `enableReinitialize` is used in this form, during the second step (platform tip step)
+             * the form values will be reset. The validate function in this form
+             * requires `amount`, `fromAccount` and `description` so we should
+             * set them as otherwise the form will not be submittable.
```

Comment:
Where would they be normally set if we were not using `enableReinitialize`?

Reply:
We would set them in the first step of the modal when the user types these values in the fields and then submit the form. For example for `fromAccount`;

https://github.com/opencollective/opencollective-frontend/blob/099b86f5968ce4b3bb56d396b4f76ef70fdba408/components/host-dashboard/AddFundsModal.js#L420

The thing is when `enableReinitalize` is not used we don't clear them out after we submit the form; when clicking `Add Funds` button (1st step submission). So the step 2 submission (adding platform tip or clicking finish button) will have access to these values.